charts/datadog/values.yaml

```
@@ -369,6 +369,15 @@ datadog:
     # datadog.systemProbe.enableOOMKill -- Enable the OOM kill eBPF-based check
     enableOOMKill: false
 
+    # datadog.systemProbe.enableRuntimeCompiler -- Enable the runtime compiler for eBPF probes
+    enableRuntimeCompiler: false
+
+    # datadog.systemProbe.mountPackageManagementDirs -- Enables mounting of various package management directories when runtime compilation is enabled
+    mountPackageManagementDirs: true
```

based on my previous comments we can convert this option to an optional list of hostPath configuration

```suggestion
    mountPackageManagementDirs: []
    ## - name: "fedora-release"
    ##   hostPath: /etc/fedora-release
    ##   mountPath: /host/etc/fedora-release
```
And base on this list, the `volumeMounts` and `volume` can be created. 

Also what can be done is to define in the `values.yaml` file all the hostPath that you want to mount by default. So like this if a customer only want one path. they can just comment the others.

please let me know if I was clear

---

Great idea, it's very clear! Thanks for the review :)